components/sysutils/smartmontools/Makefile

```diff
@@ -29,14 +29,14 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= smartmontools
 COMPONENT_SUMMARY= smartmontools contains utilities that control and monitor storage devices
-COMPONENT_VERSION= 7.1
+COMPONENT_VERSION= 7.2
 COMPONENT_REVISION=  1
```

Comment:
Please remove the COMPONENT_REVISION when you change the COMPONENT_VERSION.
Rule of thumb: If you want to change anything in a package you'll need to change either COMPONENT_VERSION (and reset COMPONENT_REVISION if there) or COMPONENT_REVISION.
If neither COMPONENT_VERSION nor COMPONENT_REVISION changes the change cannot be detected by IPS and nobody gets the update.